content/en/apps/guides/hosting/core-developer.md

```diff
@@ -0,0 +1,235 @@
+```
+
+Parts of the command:
+- `--name` creates a container called `medic-couchdb`. You can name it whatever you want, but this is how you refer to it later
+- `-e` sets an environment variable inside the container. Two are set here, for a user and password for the initial admin user.
+- `-v` maps where couchdb stores data to your local file system to ensure persistence without depending on the container, using the path *before* the `:` (the path after the colon is the internal path inside the docker image). This should be somewhere you have write access to, and want this data to be stored. The second mounted volume is for the couch configuration, which will retain settings if your container is removed. This is especially important after running the command to secure the instance (done in steps below).
+- `apache/couchdb:2` will install the latest package for CouchDB 2.x
+
+Once this downloads and starts, you will need to [initialise CouchDB](http://localhost:5984/_utils/#/setup) as noted in [their install instructions](https://docs.couchdb.org/en/2.3.1/setup/index.html#setup).
+
+You can use `docker stop medic-couchdb` to stop it and `docker start medic-couchdb` to start it again. Remember that you'll need to start it whenever you restart your OS, which might not be the case if you use a normal OS package. `docker rm medic-couchdb` will totally remove the container.
+
+Medic recommends you familiarise yourself with other Docker commands to make docker image and container management clearer.
+
+### Required environment variables
+
+Medic needs the following environment variables to be declared:
+- `COUCH_URL`: the full authenticated url to the `medic` DB. Locally this would be  `http://myadminuser:myadminpass@localhost:5984/medic`
+- `COUCH_NODE_NAME`: the name of your CouchDB's node. The Docker image default is `nonode@nohost`. Other installations may use `couchdb@127.0.0.1`. You can find out by querying [CouchDB's membership API](https://docs.couchdb.org/en/stable/api/server/common.html#membership)
+- (optionally) `API_PORT`: the port API will run on. If not defined we use `5988`
+- (optionally) `CHROME_BIN`: only required if `grunt unit` or `grunt e2e` complain that they can't find Chrome.
+
+How to permanently define environment variables depends on your OS and shell (e.g. for bash you can put them `~/.bashrc`). You can temporarily define them with `export`:
+
+```sh
+export COUCH_NODE_NAME=nonode@nohost
+export COUCH_URL=http://myadminuser:myadminpass@localhost:5984/medic
+```
+
+## Tests
+
+Refer to [the testing doc](https://github.com/medic/cht-core/blob/master/TESTING.md) in the GitHub repo.
+
+## nginx-local-ip
+
+[`nginx-local-ip`](https://github.com/medic/nginx-local-ip) is a local proxy that keeps all traffic local, and runs without latency or throttling. If sharing your local CHT instance is not required, it is the preferred method to add a valid SSL certificate (rather than `ngrok` or `pagekite`).
```

Comment:
Small wording changes
```suggestion
[`nginx-local-ip`](https://github.com/medic/nginx-local-ip) is a local proxy that keeps all traffic local, and runs without latency or throttling. If sharing your local CHT instance is not required, it is the recommended method to add a valid SSL certificate (rather than `ngrok` or similar).
```